tests/integration_tests.rs

```diff
@@ -14,6 +14,10 @@ use pod_builder::{
 use pod_setup::{wait_for_pod_complete, wait_for_pod_ready, OnFailure};
 use test_resource_manager::{TestResource, TestResourceManager, TestResourceSpec};
 
+fn in_ci_environment() -> bool {
+    std::env::var("KRUSTLET_TEST_ENV") == Ok("ci".to_owned())
```

Comment:
I think it would probably be better to use a `cargo` feature to enable/disable these tests. So then in CI we could run with `cargo test ... --features ci` and annotate the test functions with `#[cfg(feature = "ci")]`. That feels more explicit and Rust-y than an env variable to me. Could just be bike shedding here though

Reply:
Oh!  Yes!  I tried that but misunderstood the attribute syntax so it didn't work and I went back to this solution.  I agree the Cargo feature approach is nicer, but I'll defer that to a second PR.